### DISCUSSION TITLE
Analytics Options beyond Google Analytics

### DISCUSSION THREAD

**Discussion by Btibert3** [quarto-dev/quarto-web#1520] [General]
Has anyone implemented tracking analytics for their Quarto website beyond Google Analytics, and if so, how did you get that wired into your Quarto website?  I recently migrated my site from Hugo to Quarto and implemented GA4.  I am not a huge fan and would love to hear about what you are using as an alternative.

**Comment by cscheid** [comment]
I can't find a concrete example now, but I know some folks use https://plausible.io/ on their Quarto projects.

**Reply by cderv** [reply]
From Github Search : https://github.com/search?q=plausible+path%3A_quarto.yml&type=code

at least those two books:
* https://github.com/oscarbaruffa/BigBookofR/blob/4bb1fa9697f9bd5f3018e1e8cb6f0ea64882ff24/_quarto.yml#L77-L78
* https://github.com/hadley/r-pkgs/blob/d6e3d3f0bdf63f98a28a37545baff50628f1f137/_quarto.yml#L80

**Reply by mitchelloharawild** [reply]
These solutions add the tracking code only to the website's default output format, while `google-analytics` is applied at the `website` level. Is there some way to add `after-body` content across the entire website, even if a document specifies a different html output format?

E.g. a website with both regular html webpages and html slides, with tracking applied to both via a `_quarto.yml` option. Of course the appropriate a`after-body` can be added to each slide yml, but I'm hoping for a more elegant solution.

**Comment by Btibert3** [comment]
Thanks, I will take a look.
